tests/network-ovn

```diff
@@ -2127,6 +2127,57 @@ ovn_leases_tests() {
         lxc network delete lxdbr0
 }
 
+ovn_vlan_uplink_tests() {
+    echo "==> vlan uplink"
+
+    sudo ovs-vsctl set open_vswitch . \
```

Comment:
this isnt needed as its done at the top of the script.

Reply:
Removed